detekt-rules/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/bugs/IgnoredReturnValue.kt

```diff
@@ -37,13 +39,11 @@ import org.jetbrains.kotlin.types.typeUtil.isUnit
  *     if (42 == returnsValue()) {}
  *     val x = returnsValue()
  * </compliant>
+ *
+ * @configuration restrictToAnnotatedMethods - if the rule should check only annotated methods. (default: `true`)
+ * @configuration returnValueAnnotations - List of glob patterns to be used as inspection annotation (default: `["*.CheckReturnValue", "*.CheckResult"]`)
```

Comment:
```suggestion
 * @configuration returnValueAnnotations - List of glob patterns to be used as inspection annotation (default: `['*.CheckReturnValue', '*.CheckResult']`)
```

All our config file uses the single quote.